index.js

```diff
@@ -424,17 +430,36 @@ app.post('/api/xethru', Validator.body(['locationid', 'state', 'rpm', 'mov_f', '
   }
 })
 
+app.post('/api/innosent', async (request, response) => {
```

Comment:
Can you please add an Express Validator to check for the `request.body.coreid` and `request.body.data`? Is it OK if the `inPhase` and `quadrature` are `undefined`?